dashboard/app/views/admin_search/show_pilot.html.haml

```diff
@@ -1,7 +1,12 @@
 %h1
   = @pilot_name
+%h2 Enter a single email address, or several!
+%h5 Separate emails by line. For example:
```

Comment:
This line and the below don't all need to be headings, usually you don't want to use `h` tags unless you want the content to be parsed as a heading https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#example_page. This content seems more like a description, I would put this and the lines below in a `p` tag. You can use `br` tags for line breaks

Reply:
Oh nice, okay. I'll do that. Thanks!